src/OpenLoco/src/GameCommands/ClearLand.cpp

```diff
@@ -73,36 +85,117 @@ namespace OpenLoco::GameCommands
 
         World::QuarterTile qt(0xF, 0);
 
-        // TODO: for 0x00469E07 -- remove this hack when no longer needed
-        uint8_t flagStackHack[10] = { 0 };
-        flagStackHack[8] = flags;
-
-        // These are part of the parameters for 0x00469E07
-        _F003CE = pos.x;
-        _F003D0 = pos.y;
-        _F0016C = flagStackHack;
+        currency32_t cost{};
+        auto clearFunc = [flags, pos, &cost, &removedBuildings](World::TileElement& el) -> World::TileManager::ClearFuncResult {
```

Comment:
Probably best to move this out to a normal function. Bit long for a lambda imo.

Reply:
I think you've found a good compromise. The Linux CI is failing for unrelated reasons, though. Could you address that?